src/main_app/core/crypto.py

```
@@ -15,23 +18,26 @@ def _require_fernet() -> Fernet:
-        raise RuntimeError("Invalid OAUTH_ENCRYPTION_KEY format") from exc
+        key_bytes = (
+            settings.oauth.encryption_key.encode()
+            if isinstance(settings.oauth.encryption_key, str)
+            else settings.oauth.encryption_key
+        )
 
-    return _fernet
+        try:
+            _fernet = Fernet(key_bytes)
+        except ValueError as exc:
+            # Key must be a 32‑byte urlsafe base64‑encoded string
```

Ruff (0.15.14)

[warning] Comment contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)